### PR TITLE
Add MCOBS EU API collections

### PR DESCRIPTION
**Pull request by pkwasnicka-mc**
Create two new collections:
- `mcobs_eu_demo.postman_collection.json` - allows to test endpoints in MCOBS EU API
- `mcobs_eu_webhooks.postman_collection` - allows to test webhooks in MCOBS EU API

Create new environment:
- `mcobs_eu_demo.postman_environment` - designed to be used with `mcobs_eu_demo.postman_collection.json`

### CODE REVIEW

File: mcobs_eu_demo.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1638 @@
...
+							"code": 200,
+							"_postman_previewlanguage": "json",
+							"header": [
+								{
+									"key": "Content-Type",
+									"value": "application/json"
+								}
+							],
+							"cookie": [],
+							"body": "{\n    \"redirectUrl\": \"https://connectplatform.test.aiia.eu/start/storage-payments/cafc238f-2fba-4387-8a96-0a8f22c50478\",\n    \"mandateId\": \"662bea2c-a0af-42db-960f-3f7bb3cc755a\"\n}"
```
Comment (iabspiir) [inline mcobs_eu_demo.postman_collection.json]: We have the test environment for connect here in a bunch of places. Also do the other collections include sample responses?

Comment (pkwasnicka-mc) [reply mcobs_eu_demo.postman_collection.json]: Looks like no - other collection does not have sample responses. I will remove them.

**Review by iabspiir — APPROVED** [review]

**Review by iabspiir — APPROVED** [review]

File: mcobs_eu_demo.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,649 @@
+{
+	"info": {
+		"_postman_id": "b8cbacea-0202-4dd7-93fa-cde9faf00c27",
+		"name": "MCOBS EU API - demo",
```
Comment (karen-avetisyan-mc) [inline mcobs_eu_demo.postman_collection.json]: All collections shall have Mastercard at the beginning of the name.  Can you please unwrap MCOBS here and in other places.

**Comment by karen-avetisyan-mc** [comment]
This API is not live yet. Can you please hold, until API is published.

**Review by karen-avetisyan-mc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**